src/pages/CourseContent/index.js

```diff
@@ -76,6 +82,12 @@ function CourseContent(props) {
   return (
     <div className="ng-course-content">
       <div className="content">
+        <h1>{courseTitle}</h1>
+        <h2>{get(selectedExercise,"exercise.name")}</h2>
+        <ExerciseContent content={get(selectedExercise, "exercise.content")} />
+        <EditOnGithub
+          link={`${get(selectedExercise, "exercise.githubLink")}`}
+        />
```

Comment:
Try to remove that. Probably we don't need these. The below code takes care of it.